src/server/models/AreaSet.ts

```diff
@@ -1,12 +1,36 @@
-import { Generated, Insertable, Selectable, Updateable } from "kysely";
-import { AreaSetCode } from "@/__generated__/types";
+import { Generated, Insertable, Updateable } from "kysely";
+import z from "zod";
 
-export interface AreaSetTable {
-  id: Generated<number>;
-  code: AreaSetCode;
-  name: string;
+export const areaSetCodes = ["MSOA21", "OA21", "PC", "WMC24"] as const;
+
+export enum AreaSetCode {
+  MSOA21 = "MSOA21",
+  OA21 = "OA21",
+  PC = "PC",
+  WMC24 = "WMC24",
+}
```

Comment:
question: can these `const` and `enum`s be combined? `const areaSetCodes = Object.values(AreaSetCode) as const` ?

if so, could you combine the other new enums in the same way?